### DISCUSSION TITLE
Missing arm32 binary on v0.43.0 release assets

### DISCUSSION THREAD

**Discussion by osodracnai** [aquasecurity/trivy#4789] [Q&A] [answered]
### Description

The number of assets on release is dropping since v0.42.1, before that there was always 75 assets and the we've got:

- v0.42.1 with 63 assets
- v0.43.0 with 54 assets

the recent v0.43.0 misising trivy_0.43.0_Linux-ARM.tar.gz asset brakes one of my pipelines.

Is this a bug or are intentionally dropping support?

### Desired Behavior

New releases assets should have arm32 binaries

### Actual Behavior

New releases assets does not have arm32 binaries

### Reproduction Steps

```bash
check the releases page https://github.com/aquasecurity/trivy/releases
```

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

### Debug Output

```bash
N/A
```

### Operating System

N/A

### Version

```bash
N/A
```

### Checklist

- [X] Run `trivy image --reset`
- [X] Read [the troubleshooting](https://aquasecurity.github.io/trivy/latest/docs/references/troubleshooting/)

**Comment by knqyf263** [comment, marked as answer]
We dropped
https://github.com/aquasecurity/trivy/pull/4585

**Reply by dereknola** [reply]
I would like to point out that that PR only mentions removing i386 as an architecture. It makes no mention of removing ARM. This pull request https://github.com/aquasecurity/trivy/pull/4587 actually removed ARM. You should edit the wording on your description or include a mention that ARM 32bit is no longer supported.

**Reply by knqyf263** [reply]
You're right. I've updated the description. Thanks.
BTW, if you need the arm32 binary, please let us know.
